data-serving/data-service/src/model/case.ts

```diff
@@ -60,7 +60,7 @@ export const caseSchema = new mongoose.Schema(
         restrictedNotes: String,
         pathogens: [pathogenSchema],
         list: Boolean,
-        SGTF: Number,
+        SGTF: mongoose.Schema.Types.Mixed,
```

Comment:
What do you think about an enum of ['0', '1', 'NA'] and a type (below) of '0'|'1'|'NA'? On the downside the "numbers" 0 and 1 are represented as strings, on the upside the value is limited to the specific values supported.

Reply:
I like it. I made the changes and added a migration to switch SGTF from int to string.